radis/test/lbl/test_factory.py

```diff
@@ -727,62 +730,66 @@ def test_vaex_and_pandas_spectrum():
             s_vaex.get(spec_quantity)[1], s_pd.get(spec_quantity)[1], equal_nan=True
         )
 
-    #%%
-    def test_vaex_and_pandas_spectrum_noneq():
 
-        from radis import calc_spectrum
+#%%
+# Compares the Spectrum calculated under non-equilibrium conditions . Comparison is made between CONVOLUTED_QUANTITIES and dataframe df .
+def test_vaex_and_pandas_spectrum_noneq():
 
-        s, factory_s = calc_spectrum(
-            1800,
-            1820,  # cm-1
-            molecule="CO",
-            isotope="1",
-            pressure=1.01325,  # bar
-            Tgas=700,  # K
-            Tvib=710,
-            Trot=710,
-            mole_fraction=0.1,
-            wstep="auto",
-            path_length=1,  # cm
-            databank="hitemp",  # or 'hitemp', 'geisa', 'exomol'
-            optimization=None,
-            engine="vaex",
-            verbose=3,
-            return_factory=True,
-        )
+    from radis import calc_spectrum
```

Comment:
Some conditions are repeated. To improve readability and conciseness, you can store the redundant conditions with
```
conditions = {'molecule': 'CO', 'mole_fraction':0.1, 'path_length':1} #etc. 
s1 = calc_spectrum(**conditions, Trot=700, Tvib=710, Tgas=700, engine="vaex")
s2 = calc_spectrum(**conditions, Trot=700, Tvib=710, Tgas=700, engine="pandas")
```